### ISSUE TITLE
Suggestions on using rsa and elliptic-curve dependencies (failing on subtle)

### ISSUE THREAD

**Issue by markuskobler** [RustCrypto/RSA#76]
When trying to include both `rsa` and `elliptic-curve` dependencies you get a version issues with caused by `subtle` being pinned to `=2.1.1` caused by #71. What is the best path forward here to better for me to also fork and downgrade `elliptic-curve` or upgrade it for `rsa`?

```
error: failed to select a version for `subtle`.
    ... required by package `elliptic-curve v0.8.4`
    ... which is depended on by `ecdsa v0.10.2`
    ... which is depended on by `p256 v0.7.1`
...
versions that meet the requirements `=2.4.0` are: 2.4.0

all possible versions conflict with previously selected packages.

  previously selected package `subtle v2.1.1 (https://github.com/dalek-cryptography/subtle.git?tag=2.1.1#d12fc92b)`
    ... which is depended on by `rsa v0.3.0 (https://github.com/RustCrypto/RSA.git?branch=master#1d8ff05d)`
...
```

**Comment by tarcieri** [comment]
Opened #77 to try to address this

**Comment by tarcieri** [comment]
Looks like this was addressed in #74 and should no longer be a problem once there's another release including those changes.

**Comment by markuskobler** [comment]
👍 thanks
